include/z64item.h

```diff
@@ -355,7 +355,7 @@ typedef enum {
     /* 0x56 */ GI_REMAINS_GOHT,
     /* 0x57 */ GI_REMAINS_GYORG,
     /* 0x58 */ GI_REMAINS_TWINMOLD,
-    /* 0x59 */ GI_BOTTLE_POTION_RED, // ITEM_LONGSHOT
+    /* 0x59 */ GI_POTION_RED_BOTTLE, // ITEM_LONGSHOT
```

Comment:
I thought for bottles we usually put BOTTLE first.

Reply:
It would be nice to make these consistent. I do like BOTTLE first, but up to you.

Reply:
The only names to consider from `z64item.h` is really:
- `ITEM_MILK_BOTTLE` (and it's parallel `ITEM_MILK_HALF`)
- `GI_BOTTLE_POTION_RED`
- `GI_MILK_BOTTLE`
And then all the Player action which have the `_BOTTLE_` prefix

Milk bottle is weird becuase is the `MILK_` prefix prioritized or the `BOTTLE_` prefix?
And then just `GI_BOTTLE_POTION_RED`